addons/mail/static/src/web/chatter.xml

```diff
@@ -3,14 +3,14 @@
 
 <t t-name="mail.Chatter" owl="1">
     <div class="o-mail-Chatter bg-view h-100 flex-grow-1 d-flex flex-column overflow-auto" t-att-class="props.threadId === false ? 'o-chatter-disabled' : ''" t-ref="root">
-        <div class="o-mail-Chatter-topbar d-flex flex-shrink-0 flex-grow-0 pe-2">
```

Comment:
I'm a bit confused, I don't understand where this padding comes from. Anyway, I think you did well to remove it 🙂